packages/twenty-front/src/modules/workflow/search-variables/utils/isRecordOutputSchema.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  OutputSchema,
+  RecordOutputSchema,
+} from '@/workflow/search-variables/types/StepOutputSchema';
+
+export const isRecordOutputSchema = (
+  outputSchema: OutputSchema,
+): outputSchema is RecordOutputSchema => {
```

Comment:
Nope! Type guards are different than booleans. See https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types

To resume, it allows to avoid casting objects manually